core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3019,7 +3020,13 @@ If you have not fulfilled the above requirements you will lose your security dep
   For the {0} seller it is highly recommended \
 to get in contact with the {1} buyer by using the provided email address or mobile number to verify that he or she \
   is really the owner of the Zelle (ClearXchange) account.
-
+payment.fasterPayments.newRequirements.info=When using Faster Payments some banks have started verifying the receiver''s \
+  full name. Your current Faster Payments account only specifies a UK bank account number and sort code. This used to be \
+  sufficient, but may not be now for some sender''s banks. New Faster Payments accounts in Bisq now include an owner \
+  full name. Please consider recreating your account in Bisq to provide future {0} buyers with a recipient name.\n\n\
+  When recreating a Faster Payments account, take care to copy the account age verification salt from your old account \
+  and use the same sort code and account number. (The owner full name or account name need not be the same, however.) \
+  This preserves the public hash, so that the new account need not be aged or signed from scratch.
```

Comment:
```suggestion
payment.fasterPayments.newRequirements.info=Some banks have started verifying the receiver''s \
  full name for Faster Payments transfers. Your current Faster Payments account does not specify a full name.\n\n \
  Please consider recreating your Faster Payments account in Bisq to provide future {0} buyers with a full name.\n\n\
  When you recreate the account, make sure to copy the precise sort code, account number, and account age \
  verification salt values from your old account to your new account. This will ensure your existing account's age and \
  signing status are preserved.
```

Reply:
OK, I'll apply the suggested simplification.